src/protocol.rs

```diff
@@ -446,9 +446,9 @@ impl<T: InputIO> Stub<T> {
             ChangeBaudrate => {
                 let baud: ChangeBaudrateCommand = slice_to_struct(payload)?;
                 self.send_response(response);
-                self.target.delay_us(10_000); // Wait for response to be transfered
+                self.target.delay_us(15_000); // Wait for response to be transfered
                 self.target.change_baudrate(baud.old, baud.new);
-                self.send_greeting();
```

Comment:
Why is this being deleted?

Reply:
Looking at the C stubs, [greeting is only sent at the start](https://github.com/espressif/esptool/blob/6c5cfd6dabcdc04397589dae160db88b866fd791/flasher_stub/stub_flasher.c#L488) and here is how the `ESP_SET_BAUD` is processed: https://github.com/espressif/esptool/blob/6c5cfd6dabcdc04397589dae160db88b866fd791/flasher_stub/stub_flasher.c#L379 https://github.com/espressif/esptool/blob/6c5cfd6dabcdc04397589dae160db88b866fd791/flasher_stub/stub_flasher.c#L266